### DISCUSSION TITLE
Add VS Code support for a rewrap or reflow action for comments

### DISCUSSION THREAD

**Discussion by juliasilge** [quarto-dev/quarto-cli#9325] [Feature Requests]
### Description

If I have a Quarto file like this:

````md
---
title: "About"
---

It is a truth universally acknowledged, that a single man in possession of a good fortune must be in want of a wife.

```{r}
# It is a truth universally acknowledged, that a single man in possession of a good fortune must be in want of a wife.
# 
# However little known the feelings or views of such a man may be on his first entering a neighbourhood, this truth is so well fixed in the minds of the surrounding families, that he is considered as the rightful property of some one or other of their daughters.
# 
# "My dear Mr. Bennet," said his lady to him one day, "have you heard that Netherfield Park is let at last?"
# 
# Mr. Bennet replied that he had not.
# 

1 + 1 
```
````

then in RStudio, I can use "Reflow comment" to break up the comments into my chosen line length. I see that VS Code has [this extension](https://open-vsx.org/extension/stkb/rewrap) but it does not work at all in a Quarto document (does nothing).

**Comment by mcanouil** [comment]
VSCode extension (and Visual Editor) repository is <https://github.com/quarto-dev/quarto>.
Better to file feature requests (or bug reports) regarding those tools directly on the repository.

**Reply by juliasilge** [reply]
Added here: https://github.com/quarto-dev/quarto/issues/408
